cypress/integration/test/testimonials.spec.js

```diff
@@ -0,0 +1,112 @@
+const endpoint = 'http://localhost:3000/testimonials';
+const id = 1
+const errorID = 100
+
+const testimonie = {
+    "name": "cypress test",
+    "image": "cypress.jpg",
+    "content": "content cypress test"
+}
+
+const errorTestimonie = {
+    "image": "cypress.jpg",
+    "content": "content cypress test"
+}
+
+const testimonieUpdate = {
+    "name": "cypress test update",
+    "image": "cypress.jpg update",
+    "content": "content cypress test update"
+}
+
+const addTestimonie = testimonie => {
+    cy.request('POST', endpoint, testimonie)
+    expect(testimonie).to.have.all.keys('name', 'image', 'content')
+}
+
+describe('TESTIMONIALS TESTING', () => {
+    
+    it("GET - Show Testimonials Paginated", () => {
+
+        cy.request('GET', endpoint)
+        .then((response) => {
+            expect(response.status).equal(200);
+            expect(response.body).to.not.be.null;

```

Comment:
Seba esta bastante bien en general. Cambiaria el expect(X).to.not.be.null por un expext(X).to.deep.equal(X) es decir, no testear por lo que no es, sino por lo que se espera.

Reply:
Listo Maxi, cambio realizado